webpack.config.js

```diff
@@ -110,7 +110,12 @@ module.exports = (env) => ({
                               attribute: 'stroke',
                               value: '#E97025',
                               className: 'fi-icon-component-highlight-stroke'
-                            }
+                            },
+                            {
+                              attribute: 'fill',
+                              value: '#00347A',
+                              className: 'fi-icon-component-brand-fill'
+                            },
```

Comment:
Do we need this?

Reply:
I think the designers might have an answer to this one. If there's need to be able to change the color, then this is the way to go. If not, we can leave this out.

Reply:
Seems like the option is wanted. Not quite sure if the `brand` is a necessary word here, but on the other hand I can't come up with a better one off the top of my head.